addons/mail_group/models/mail_group_moderation.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+from odoo.tools import email_normalize
+
+
+class MailGroupModeration(models.Model):
+    """Represent the moderation rules for an email address in a group."""
+
+    _name = 'mail.group.moderation'
+    _description = 'Mailing List black/white list'
+
+    email = fields.Char(string='Email', required=True)
```

Comment:
I would override create / write to ensure we only store normalized emails here. That way all comparisons are easy to do. A bit like what we did with blacklist model, whatever you give, only normalized emails are stored.

Reply:
Good idea :)

Reply:
In the end it will be tricky to return existing record in the create (will need to build a SQL query depending on the input), so I'll just normalize the email in create / write